coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.Development.json

```diff
@@ -1,9 +1,10 @@
 {
   "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+      "LogLevel": {
+          "Default": "Information",
+          "Microsoft": "Warning",
+          "Microsoft.Hosting.Lifetime": "Information"
+          
```

Comment:
Please make sure not to commit any changes to the appsettings.Development.json

Reply:
accidentally committed the changes and later removed.